src/wizard/flutter/index.md

```diff
@@ -88,4 +88,20 @@ Future<void> processOrderBatch(ISentrySpan span) async {
 }
 ```
 
-Check out [the documentation](https://docs.sentry.io/platforms/flutter/performance/instrumentation/) to learn more about the API and automatic instrumentations.
+Check out [the documentation](/platforms/flutter/performance/instrumentation/) to learn more about the API and automatic instrumentations.
+
+## Debug Symbols
+
+Debug symbols provide information that Sentry displays on the **Issue Details** page to help you triage an issue. We offer a range of methods to provide Sentry with debug symbols.
+
+With default settings, complete stack traces are available in your Dart error, out of the box, unless you use [`split-debug-info`](https://flutter.dev/docs/perf/app-size#reducing-app-size) and [`obfuscate`](https://flutter.dev/docs/deployment/obfuscate). In those cases, you need to [upload the debug information files](/platforms/flutter/upload-debug/) generated by the `flutter` build, so Sentry can show you proper stack traces.
+
+For iOS, macOS and Android NDK native crashes, it is required to [upload the debug symbols](/platforms/flutter/upload-debug/) generated by the `flutter` build.
+
+## Source Context
+
+If Sentry has access to application source code, it can show snippets of code around the location of stack frames. We refer to these snippets as _source context_.
+
+Source context can be very useful for quickly identify problematic code.
```

Comment:
```suggestion

If Sentry has access to your application's source code, it can show snippets of code (_source context_) around the location of stack frames, which helps to quickly pinpoint problematic code. 


```